spec/models/candidate_interface/contact_details_form_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe CandidateInterface::ContactDetailsForm, type: :model do
+  let(:data) do
+    {
+      phone_number: '07700 900 982',
+    }
+  end
+
+  describe '.build_from_application' do
+    it 'creates an object based on the provided ApplicationForm' do
+      application_form = FactoryBot.build_stubbed(:application_form, data)
+      contact_details = CandidateInterface::ContactDetailsForm.build_from_application(
+        application_form,
```

Comment:
nitpick, do we need the final comma here? it does not seem a list of items with high chance to grow

Reply:
T͙̟̠ͮ̔̔h̤͙̮͔̠͇͉Ĕ̖ ̱͇͑ͪr͕̥͚͙̰̼̉̑ͯ̎͊̅Ub̩͇̤̲̦̥̤O͚͉͍͎̿ͩͤ̔c̪̼͈̣ͩ̋ͭ̂O̳̗̘̫ͬ̇̚̚p ͍̮͇̱̩̹̑͗͐̉ͯ̂Iṋ͓̘̩̬͕͎S͚̥̿̒i̅ͭS̩̘̭͓t̖͆S̝̬͎̙͎̍̍̑͋̎



(... there's a linting rule that forces us to add it)